notebooks/prometheus2_evaluation.ipynb

```diff
@@ -0,0 +1,12347 @@
+{
```

Comment:
<p>Maybe this can be removed or commented and with a comment added that if the user ran the RAG pipeline no need for this step</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/92/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>Makes sense. I will clarify this point.</p>